src/main/java/pl/magisterka/dominikszojda/repository/DogRepository.java

```diff
@@ -1,6 +1,25 @@
 package pl.magisterka.dominikszojda.repository;
 
-public class DogRepository {
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+import pl.magisterka.dominikszojda.entity.DogEntity;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+@Repository
+public interface DogRepository extends CrudRepository<DogEntity, UUID>{
+
+
+    Optional<DogEntity> findByNameAndAge(String name, int age);
+
+
+    List<DogEntity> findAll();
+
+
+    Optional<DogEntity> findDogById(UUID UUID);
```

Comment:
nazwy parametrów z małej litery: `UUID uuid`
```suggestion
    Optional<DogEntity> findDogById(UUID uuid);
```

Reply:
porpawione